cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh

```diff
@@ -20,4 +20,5 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
+# shellcheck disable=SC2086
```

Comment:
Please add a more detailed comment

Reply:
Is that ok?

Reply:
If they produce the same result please change it to "${FLUENTD_ARGS}" and remove the lint exception.

Reply:
Has been completed